index.js

```diff
@@ -1,12 +1,31 @@
 'use strict';
 
+// If `to` has properties that `from` does not have, remove them
+const removeProperty = (to, from, property) => {
+	if (property in from) {
```

Comment:
This check is inconsistent with all the other checks that check own keys. You probably want `if (from.hasOwnProperty(property) {` here.